odl/discr/lp_discr.py

```diff
@@ -785,6 +997,60 @@ def uniform_discr(min_corner, max_corner, nsamples,
                                    **kwargs)
 
 
+def sequence_space(shape, exponent=2.0, impl='numpy', **kwargs):
+    """Return an object mimicing the sequence space ``l^p(R^d)``.
```

Comment:
I mean the space of functions on ALL the integers (similar to how our FunctionSpace(IntervalProd) is a space of functions on ALL reals in some set). Discretization would then be perhaps truncation.